src/utils/operations.utils.spec.ts

```diff
@@ -0,0 +1,30 @@
+import {
```

Comment:
Mock a timezone equal to UTC to prevent pipeline failures
This may be helpful https://stackoverflow.com/questions/56261381/how-do-i-set-a-timezone-in-my-jest-config

Reply:
hardcoded timestamp, got another way to fix